### DISCUSSION TITLE
Change Clickable instead of id.

### DISCUSSION THREAD

**Discussion by JenuelDev** [laravel/nova-issues#3347] [Q&A] [answered]
So, for the default to view an item, we can click the id or the eye icon. But how do add a different column?  
In Here I would like to add a click function to view item, by clicking the name of the car.
![image](https://user-images.githubusercontent.com/31676163/114833941-aaf15d00-9e02-11eb-9510-480b8b6983a6.png)

**Comment by alexrififi** [comment, marked as answer]
You can use https://github.com/dcasia/clickable-table-row
